airbyte-webapp/src/components/base/Input/Imput.module.scss

```diff
@@ -0,0 +1,11 @@
+.visibilityButton {
```

Comment:
Nitpick:
Typo in filename - `components/base/Input/Imput.module.scss`

Reply:
fixed